src/main/java/seedu/address/ui/MeetingCard.java

```diff
@@ -0,0 +1,74 @@
+package seedu.address.ui;
+
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+import javafx.scene.layout.Region;
+import seedu.address.model.meeting.Meeting;
+
+/**
+ * An UI component that displays information of a {@code Meeting}.
+ */
+public class MeetingCard extends UiPart<Region> {
+
+    private static final String FXML = "MeetingListCard.fxml";
+
+    /**
+     * Note: Certain keywords such as "location" and "resources" are reserved keywords in JavaFX.
+     * As a consequence, UI elements' variable names cannot be set to such keywords
+     * or an exception will be thrown by JavaFX during runtime.
+     *
+     * @see <a href="https://github.com/se-edu/addressbook-level4/issues/336">The issue on AddressBook level 4</a>
+     */
+
+    public final Meeting meeting;
+    @FXML
+    private Label id;
+    @FXML
+    private Label title;
+    @FXML
+    private Label dateTime;
+    @FXML
+    private Label attendees;
+    @FXML
+    private Label description;
+    @FXML
+    private Label meetingLocation;
```

Comment:
small nit here... could we reorder these 2?
```suggestion
    @FXML
    private Label meetingLocation;
    @FXML
    private Label description;
```